Vagrantfile

```diff
@@ -27,8 +26,7 @@ $bootstrap = <<BOOTSTRAP
   sudo chown -R vagrant /var/lib/
   sudo chown -R mysql /var/log/mysql
   sudo chown -R mysql /var/log/mysql
-  sudo apt-fast -y install emacs24-nox
-  sudo apt-fast -y install libmagickwand-dev
+  sudo apt-fast -y install emacs24-nox findutils realpath libmagickwand-dev
```

Comment:
On 09/01/2016 10:47 PM, Jon Kiparsky wrote:
> In Vagrantfile
> <https://github.com/bethlakshmi/GBE2/pull/870#discussion_r77286459>:
>
>> @@ -27,8 +26,7 @@ $bootstrap = <<BOOTSTRAP
>>    sudo chown -R vagrant /var/lib/
>>    sudo chown -R mysql /var/log/mysql
>>    sudo chown -R mysql /var/log/mysql
>> -  sudo apt-fast -y install emacs24-nox
>> -  sudo apt-fast -y install libmagickwand-dev
>> +  sudo apt-fast -y install emacs24-nox findutils realpath libmagickwand-dev
>
> What happens if one of these installs fails? Do the rest go through, or
> does the whole line fail, or do we exit on failure, or what?

The rest go through.  I replaced:

   sudo apt-fast -y install emacs24-nox

with:

   sudo apt-fast -y install argle-bargle

on an unbuilt Vagrantfile, up'ed it, and it spewed red errors about the 
package not found, and continued.

After that, I cleared argle-bargle, and replaced it with:

   sudo apt-fast -y install realpath looseutils libmagickwand-dev

because if there is a utility package to help you find files, there 
should also be one that helps you loose them, but there is not.  Virtual 
Box and Vagrant failed to install realpath and libmagickwand-dev, but 
only reported not being able to find looseutils.  Surprising, I was 
expecting more apt-get like behavior.  Separating them over three lines. 
  Good catch.

Hunter